src/Gaufrette/Adapter/PhpseclibSftp.php

```diff
@@ -2,11 +2,16 @@
 
 namespace Gaufrette\Adapter;
 
+@trigger_error('The '.__NAMESPACE__.'\PhpseclibSftp is deprecated since version 0.4. Use Gaufrette\Adapters\Phpseclib\Sftp instead.', E_USER_DEPRECATED);
+
 use Gaufrette\Adapter;
 use phpseclib\Net\SFTP as SecLibSFTP;
 use Gaufrette\Filesystem;
 use Gaufrette\File;
 
+/**
+ * @deprecated 0.4 This adapter is deprecated since version 0.4. Use Gaufrette\Adapters\Phpseclib\Sftp instead.
+ */
 class PhpseclibSftp implements Adapter,
```

Comment:
I think the old classe should extends from the new one to not duplicates the code.